e2e/cmd/e2e.go

```diff
@@ -573,18 +531,25 @@ func installRelease(_ context.Context, s scenario.Scenario, logger *logrus.Logge
 		fmt.Sprintf("daemonset.clusterVersion=%s.%s.x", versionSplitted[0], versionSplitted[1]),
 	)
 
-	o, err := helm.InstallRelease(filepath.Join(dir, cliArgs.NrChartPath), cliArgs.Context, logger, options...)
+	releaseName := randString(5)
```

Comment:
You can probably use https://pkg.go.dev/k8s.io/apimachinery/pkg/util/rand#String